WikiDrips/UImageExtensions.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  TitleImageView.swift
+//  WikiDrips
+//
+//  Created by Bart Cullimore on 4/6/17.
+//  Copyright © 2017 Bart Cullimore. All rights reserved.
+//
+
+import UIKit
+
+extension UIImage {
+    
+    func image(withInitials: String?, in bounds: CGRect) -> UIImage {
```

Comment:
You can add an internal variable name so you don't have a weird `withInitials` variable name:
`func image(withInitials initials: String?,....`